services/src/test/scala/cromwell/services/instrumentation/impl/statsd/StatsDInstrumentationServiceActorSpec.scala

```diff
@@ -47,19 +47,18 @@ class StatsDInstrumentationServiceActorSpec extends TestKitSuite with FlatSpecLi
     super.afterAll()
   }
 
-  case class StatsDTestBit(description: String, metric: CromwellMetric, expectedPackets: Set[String])
+  // Fuzzy packets are metrics which we expect to see but for which cannot determine the exact values for asynchronous reasons
+  case class StatsDTestBit(description: String, metric: CromwellMetric, expectedExactPackets: Set[String], expectedFuzzyPackets: Set[String] = Set.empty)
```

Comment:
```suggestion
  final case class StatsDTestBit(description: String, metric: CromwellMetric, expectedExactPackets: Set[String], expectedFuzzyPackets: Set[String] = Set.empty)
```

Reply:
Is this just because you want to play with the new github feature @geoffjentry ? 😄

Reply:
haha no, i went to make this comment and it suggested I use the feature. Totally paid off!